src/test/scala/viper/gobra/chopper/SCCUnitTests.scala

```diff
@@ -9,8 +9,8 @@ package viper.gobra.chopper
 import org.scalatest.funsuite.AnyFunSuite
 import org.scalatest.Inside
 import org.scalatest.matchers.should.Matchers
-import viper.gobra.util.ViperChopper
-import viper.gobra.util.ViperChopper.SCC.Component
+import viper.silver.ast.utility.Chopper
+import viper.silver.ast.utility.Chopper.SCC.Component
 
 class SCCUnitTests extends AnyFunSuite with Matchers with Inside {
```

Comment:
We can remove this file since Viper contains a bigger test file that subsumes this one to my knowledge.